<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SWATool</class>
 <widget class="QMainWindow" name="SWATool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 217, 217);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(ValeoLogo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(23, 60, 116);</string>
    </property>
    <property name="text">
     <string>Transfer Plan tool</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1-Plarion Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2-Plarion Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>7- Starting Rolling Build</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_User">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_Pass">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Polarion_Plan">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Run">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(55, 110, 212);
background-color: rgb(241, 241, 241);</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>431</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="Start_RB">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3-Project</string>
    </property>
   </widget>
   <widget class="QComboBox" name="P_Project">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 248, 244);</string>
    </property>
    <item>
     <property name="text">
      <string>VW_MEB_Inverter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BMW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PSA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CEVT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100-KW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>model_kit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAI</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Version">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>450</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Version :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6-Polarion SW Plan</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Int_Plan_Document">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);</string>
    </property>
   </widget>
   <widget class="QLabel" name="User_Mark">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pass_Mark">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Project_Mark">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Int_Plan_Mark">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Polarion_SW_Plan_Mark">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RB_Mark">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5-Integration Plan Doc.</string>
    </property>
   </widget>
   <widget class="QLabel" name="Warning_Message">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Brows..</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>8- Local Repo Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="L_Repo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SWA_Baseline">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4-SWA Baseline.</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Logo.qrc"/>
 </resources>
 <connections/>
</ui>
